boilerplates/react/files/$vite.config.ts.ts

```diff
@@ -12,5 +19,41 @@ export default async function getViteConfig(props: TransformerProps) {
     options: reactOptions,
   });
 
+  if (props.meta.BATI.has("sentry")) {
+    // load app level environment variables from `.env` file
```

Comment:
@magne4000 I currently stuck with transforming existing vite.config code. Have a look at my code maybe you can help me to solve this. 
An alternative solution would be, that the wrapper exists in the default function. I had to change the exports in `packages/core/src/magicast.ts` to access the needed functions to manipulate the AST.

Reply:
> I had to change the exports in `packages/core/src/magicast.ts` to access the needed functions to manipulate the AST.

👍 no issue with that

> An alternative solution would be, that the wrapper exists in the default function

I already tried that, but `magicast` doesn't support this syntax sadly, so even if you could generate one, further `$vite.config.ts.ts` wouldn't be able to parse it.

When faced with the same problem last time, my solution was to use `process.ENV` instead of `mode` so I can avoid creating the function.
Then complex manipulations where directly done in original [`vite.config.ts`](https://github.com/vikejs/bati/blob/main/boilerplates/shared/files/vite.config.ts).